File: src/main/java/io/AppiumManager.java
package io;

import io.appium.java_client.AppiumDriver;
import utils.LocProperties;

public class AppiumManager {

    protected static final String driverURL = LocProperties.getProperties().getProperty("appiumURL");
    protected static AppiumDriver driver = null;
    protected static final int implicitWait = 5;

}

File: src/main/kotlin/io/AppiumManager.kt
package io

import io.appium.java_client.AppiumDriver
import utils.LocProperties
import java.io.File

open class AppiumManager {

    protected companion object {
        val driverURL = LocProperties.getProperty("appiumURL")
        val implicitWait: Long = 5
        var app: File? = null
        var driver: AppiumDriver? = null
    }
}